policy/modules/admin/sudo.if

```diff
@@ -35,6 +35,17 @@ template(`sudo_role_template',`
 		attribute sudodomain;
 	')
 
+	## <desc>
+	##	<p>
+	##	Determine whether the sudo domain
+	##	can connect to TCP HTTP ports. This
+	##	is needed if an additional authentication
+	##	mechanism via an HTTP server is
+	##	required for users to use sudo.
+	##	</p>
+	## </desc>
+	gen_tunable(sudo_$1_tcp_connect_http_port, false)
```

Comment:
Why does this need to be per-role, rather than all or nothing?

Reply:
It really doesn't need to be. I figured that the flexibility of making it per-role could be valuable. If not, I don't object to removing this.

Reply:
Lets start with a single tunable and if that's too limiting, we can always make it more granular.